pkg/reconciler/configuration/resources/revision_test.go

```diff
@@ -78,6 +78,7 @@ func TestMakeRevisions(t *testing.T) {
 					serving.ConfigurationLabelKey:           "build",
```

Comment:
you don't seem to have any positive tests here.

Reply:
yep - on my todo list - wanted to get the core of this out there for review ASAP. Coming soon...

Reply:
done